instrument/ARGUS_Definition_remapped.xml

```diff
@@ -0,0 +1,501 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- For help on the notation used to specify an Instrument Definition File 
+     see http://www.mantidproject.org/IDF -->
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
+ name="ARGUS" valid-from   ="1995-01-31 23:59:59"
```

Comment:
I'm a little confused at the overlap between these valid dates here and the ones in `ARGUS_Definition.xml`?